src/omero_cli_render.py

```diff
@@ -254,26 +238,26 @@ def _configure(self, parser):
         sub = parser.sub()
         info = parser.add(sub, self.info, DESC["INFO"])
         copy = parser.add(sub, self.copy, DESC["COPY"])
+        set = parser.add(sub, self.set, DESC["SET"])
```

Comment:
This variable overrides the built-in Python `set` object. It works, but something to beware of in future e.g. if this function is extended.

Reply:
Generally better to avoid overriding, pushed a commit.